sdk/cosmos/azure-cosmos/azure/cosmos/_retry_utility.py

```diff
@@ -76,6 +78,16 @@ def Execute(client, global_endpoint_manager, function, *args, **kwargs):
     timeout_failover_retry_policy = _timeout_failover_retry_policy._TimeoutFailoverRetryPolicy(
         client.connection_policy, global_endpoint_manager, *args
     )
+    # HttpRequest we would need to modify for Container Recreate Retry Policy
+    request: Optional[HttpRequest] = None
+    if args and len(args) > 3:
+        # Reference HttpRequest instance in args
+        request = args[3]
+        container_recreate_retry_policy = _container_recreate_retry_policy.ContainerRecreateRetryPolicy(
+            client, client._container_properties_cache, request, *args)
+    else:
```

Comment:
Curious: when request will be None?

Reply:
In this case during parts of queries args will be empty.